tests/.oomph-cache/.../run_at_exit_too_many_error: too many run_at_exit() calls
Program exited with status 1
